### PR TITLE
Refactor ops reports for Tock truancy out of Angry Tock

### LINKED ISSUE #480: Refactor Angry Tock so messages to supes/op are handled in a separate script
The 18F Director of Operations depends on the list of Tock truants in order to make sure timecards are completed and billing can be handled properly. Since that functionality is operationally important, it should probably not be combined with functionality that is merely convenient.

Furthermore, we might want to add an additional test for when the new ops-messaging script is modified, reminding the author of the changes to check with the director of operations before merging.

### PR DESCRIPTION
**Pull request by mgwalker**
- adds a new date utility for getting the days of the current working week (and leaves out days that are holidays)
- removes ops report from Angry Tock
- updates Angry Tock to use date utility to schedule its shouting matches
- adds a new script for ops reporting
- adds a GitHub action that detects changes to the new ops reporting script and reminds PR authors to collaborate with the 18F director of operations
- closes #480 

---

Checklist:

- [x] Code has been formatted with prettier
- [x] N/A ~The [OAuth](https://github.com/18F/charlie/wiki/OAuthEventsAndScopes) wiki
      page has been updated if Charlie needs any new OAuth events or scopes~
- [x] The [Environment Variables](https://github.com/18F/charlie/wiki/EnvironmentVariables)
      wiki page has been updated if new environment variables were introduced
      or existing ones changed
- [x] The dev wiki has been updated, e.g.:
  - local development processes have changed
  - major development workflows have changed
  - internal utilities or APIs have changed
  - testing or deployment processes have changed
- [x] N/A ~If appropriate, the NIST 800-218 documentation has been updated~

### CODE REVIEW

**Comment by mgwalker** [comment]
Wiki updates are in the wiki branch `482` and can be merged after this PR is merged.

**Review by geekygirlsarah — APPROVED** [review]
I think it's all there and looks good!

**Pull request merged**
